protocol/forks/hf-20230515.md

```diff
@@ -0,0 +1,12 @@
+# HF-20230515
+
+The May 2023 hard fork is the second upgrade to be comprised of [CHIPs](#chips).
+
+## Contents
+
+The May 2023 hard fork is comprised the following CHIPs:
+
+1. [CHIP 2021-01 Allow Transactions to be smaller in size](https://gitlab.com/bitcoin.cash/chips/-/blob/master/CHIP-2021-01-Restrict%20Transaction%20Versions.md)
```

Comment:
FWIW, these links should probably all point to documents within the spec repo itself instead of an external gitlab.

Reply:
If making a copy of a chip, would need to document the commit at which the copy was made + additional link to the outside spec?

Could inject a sub-repo (whatever those are called) maybe... :grimacing: